r/vignettes/developing.Rmd

```diff
@@ -38,6 +39,14 @@ set -e
 set -x
 ```
 
+
+```{bash, save=run & windows, hide=TRUE}
+# Part of Path, including the path to R's bin, needs to be fixed.
+export PATH=$(cygpath --path $PATH)
+# For some reason CRAN Mirror goes missing in CI
+echo 'options(repos=structure(c(CRAN="https://cloud.r-project.org")))' > $HOME/.Rprofile
```

Comment:
That's odd! What happens when this line isn't here? Do you have a link to a build where that was an issue?

Reply:
Yeah just got this:
```
Error in contrib.url(repos, "source") : 
  trying to use CRAN without setting a mirror
```

in this run: https://github.com/ursacomputing/crossbow/runs/4172831531?check_suite_focus=true

Reply:
Ah, yeah, we probably should set a more general cran mirror in this build, but feel free to make a follow-on Jira for that